src/phoenix/server/app.py

```diff
@@ -150,6 +154,28 @@
 _Callback: TypeAlias = Callable[[], Union[None, Awaitable[None]]]
 
 
+def import_object_from_file(file_path: str, object_name: str) -> Any:
+    """Import an object (class or function) from a Python file."""
+    try:
+        if not os.path.isfile(file_path):
+            raise FileNotFoundError(f"File '{file_path}' does not exist.")
+        module_name = f"custom_module_{hash(file_path)}"
+        spec = importlib.util.spec_from_file_location(module_name, file_path)
```

Comment:
Would someone be able to define a small package this way, or only a file?

Reply:
I think we should 1: stick to one approach for now, and files will be simpler to implement as users don't need all of the package boilerplate